### PR TITLE
accounts/usbwallet: fix staticcheck warnings

### PR DESCRIPTION
**Pull request by gballet**

### CODE REVIEW

File: accounts/usbwallet/ledger.go [hunk 1]
```diff
@@ -162,8 +162,8 @@ func (w *ledgerDriver) SignTx(path accounts.DerivationPath, tx *types.Transactio
 		return common.Address{}, nil, accounts.ErrWalletClosed
 	}
 	// Ensure the wallet is capable of signing the given transaction
-	if chainID != nil && w.version[0] <= 1 && w.version[1] <= 0 && w.version[2] <= 2 {
-		return common.Address{}, nil, fmt.Errorf("Ledger v%d.%d.%d doesn't support signing this transaction, please update to v1.0.3 at least", w.version[0], w.version[1], w.version[2])
+	if chainID != nil && w.version[0] <= 1 && w.version[2] <= 2 {
+		return common.Address{}, nil, fmt.Errorf("ledger v%d.%d.%d doesn't support signing this transaction, please update to v1.0.3 at least", w.version[0], w.version[1], w.version[2])
```
Comment (karalabe) [inline accounts/usbwallet/ledger.go]: It's a name, that's why it's capitalized

File: accounts/usbwallet/ledger.go [hunk 2]
```diff
@@ -162,8 +162,8 @@ func (w *ledgerDriver) SignTx(path accounts.DerivationPath, tx *types.Transactio
 		return common.Address{}, nil, accounts.ErrWalletClosed
 	}
 	// Ensure the wallet is capable of signing the given transaction
-	if chainID != nil && w.version[0] <= 1 && w.version[1] <= 0 && w.version[2] <= 2 {
-		return common.Address{}, nil, fmt.Errorf("Ledger v%d.%d.%d doesn't support signing this transaction, please update to v1.0.3 at least", w.version[0], w.version[1], w.version[2])
+	if chainID != nil && w.version[0] <= 1 && w.version[2] <= 2 {
```
Comment (karalabe) [inline accounts/usbwallet/ledger.go]: Why did you remove the version[1] check?

Comment (gballet) [reply accounts/usbwallet/ledger.go]: Because `staticcheck` complains that a `byte` is always greater than 0.

Comment (karalabe) [reply accounts/usbwallet/ledger.go]: Ah, good point :P

File: accounts/usbwallet/ledger.go (hunk 1 above)
Comment (gballet) [reply accounts/usbwallet/ledger.go]: ok, but you did not capitalize it on lines 65 and 69 either. Just let me know what you prefer, I can change either way, although the capital L will force us to add more `//lint:ignore` lines

Comment (karalabe) [reply accounts/usbwallet/ledger.go]: Lines 65 and 69 are internal errors used by Geth. This one is actually show to the user on the console.
